plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -237,8 +242,23 @@ func init() {
 	})
 }
 
-func createRealIngestor(client localClient, database string, tableName string) (localIngestor, error) {
-	ingestor, err := ingest.New(client.(*kusto.Client), database, tableName, ingest.WithStaticBuffer(bufferSize, maxBuffers))
+// For each table create the ingestor
+func createIngestorByTable(client *kusto.Client, database string, tableName string, ingestionType string) (ingest.Ingestor, error) {
+	var ingestor ingest.Ingestor
+	var err error
+	if strings.ToLower(ingestionType) == managedIngestion {
+		mi, err := ingest.NewManaged(client, database, tableName)
```

Comment:
We do not pass the static buffer to the queued ingest client inside of the managed client - how does that affect memory consumption?

Reply:
Good question @AsafMah , we found the change was not much if we pass the static buffer. Do we want to pass the static buffer in either cases (managed and queued)

Reply:
We'll keep it like this for now.